config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -226,18 +226,19 @@ presets:
     value: 50
   - name: PERF_TESTS_PRINT_COMMIT_HISTORY
     value: true
+  - name: LOG_DUMP_SAVE_SERVICES
+    value: "containerd"
 
   ###### Scalability Envs
   ### Common env variables for containerd scalability-related suites.
+  # Deprecated. Should be removed after 1.20 branch will be removed from testing
```

Comment:
We cannot remove the preset, but we can remove the CL2_PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT, right?

Reply:
I do not think so? I guess that mitigation `net.netfilter.nf_conntrack_tcp_be_liberal=1` is not backported to branches 1.19 and 1.20. Is this correct?

Reply:
Let's set it only in periodics and presubmits then maybe:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml#L294

I would really like to avoid having that in master jobs.

Reply:
In this PR I'm removing `preset-e2e-scalability-containerd: "true"` from all master jobs, effectively removing CL2_PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT from all master jobs.

I do not think that we need CL2_PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT in presubmits/periodics. Could you elaborate on that?

Reply:
OK - you're right, it's only in this preset.

So yeah - I'm fine with this PR.